libs/frontend/feature-build/src/components/toolbar/ToolbarColorPicker.vue

```diff
@@ -38,7 +48,7 @@ withDefaults(
 )
 const emit = defineEmits<{
   (e: 'click', event: Event): void
-  (e: 'selectColor', color: IRgba): void
-  (e: 'applyGradient', gradient: string): void
+  (e: 'selectColor', color: IPickerColor): void
+  (e: 'applyGradient', gradient: IThemedGradient): void
```

Comment:
I pass `IThemedGradient` everywhere instead of `string`, the only difference is it also stores the un-resolved value (with theme variables).

This was done before I added `resolveThemeVar` to the `ColorPicker` props, so it may be possible to compute it on the fly, instead of storing both values 🤔